app/src/main/java/org/wikipedia/login/LoginClient.kt

```diff
@@ -38,10 +38,11 @@ class LoginClient {
                     }
                     cb.success(loginResult)
                 } else if (LoginResult.STATUS_UI == loginResult.status) {
+                    val parsedMessage = loginResult.message?.let { ServiceFactory.get(wiki).parseText(it) }?.text
```

Comment:
Would it be a good idea that we add a generic error message if the `parseText` is `null`?

Reply:
Sure, we can fall back to the unparsed message in case parsing fails for whatever reason.